### PR TITLE
Document source_time_zone flag in MoveTables

### PR DESCRIPTION
**Pull request by rohit-nayak-ps**
Document the new `source_time_zone` flag added in https://github.com/vitessio/vitess/pull/10102

Signed-off-by: Rohit Nayak <rohit@planetscale.com>

### CODE REVIEW

File: content/en/docs/14.0/reference/vreplication/movetables.md [hunk 1]
```diff
@@ -196,6 +200,29 @@ Usually, any routing rules created by the workflow in the source and target keys
...
+**optional**\
+**default** ""
+
+<div class="cmd">
+
+Specifying this flag causes any DATETIME fields to be converted from given time zone into UTC. It is expected that the application has
+stored *all* DATETIME fields, in all tables being moved, in the specified time zone. On the target these datetimes will be stored in UTC.
+
+As a best practice, Vitess expects users to run their MySQL servers in UTC. So we do not specify a target time zone for the conversion.
+It is expected that the time zone tables are loaded on the target mysql servers (for example, by executing the script generated by `mysql_tzinfo_to_sql /usr/share/zoneinfo`).
```
Comment (mattlord) [inline content/en/docs/14.0/reference/vreplication/movetables.md]: I would instead use:
```
It is expected that the [time zone tables have been pre-populated](https://dev.mysql.com/doc/refman/en/time-zone-support.html#time-zone-installation) on the target mysql servers.
```

File: content/en/docs/14.0/reference/vreplication/movetables.md [hunk 2]
```diff
@@ -196,6 +200,29 @@ Usually, any routing rules created by the workflow in the source and target keys
 
 </div>
 
+#### -source_time_zone
+**optional**\
+**default** ""
+
+<div class="cmd">
+
+Specifying this flag causes any DATETIME fields to be converted from given time zone into UTC. It is expected that the application has
```
Comment (mattlord) [inline content/en/docs/14.0/reference/vreplication/movetables.md]: Nit, but I would put special words like DATETIME in all caps and within bacticks as a way to denote their special meaning: `DATETIME`

In the next sentence we use "datetimes", we could instead use "`DATETIME` values"

File: content/en/docs/14.0/reference/vreplication/movetables.md [hunk 3]
```diff
@@ -196,6 +200,29 @@ Usually, any routing rules created by the workflow in the source and target keys
...
+
+As a best practice, Vitess expects users to run their MySQL servers in UTC. So we do not specify a target time zone for the conversion.
+It is expected that the time zone tables are loaded on the target mysql servers (for example, by executing the script generated by `mysql_tzinfo_to_sql /usr/share/zoneinfo`). 
+
+Any reverse replication streams running after a SwitchWrites will do the reverse date conversion on the source.
+
+Note that selecting the datetime columns from the target will now give the times in UTC. It is expected that the application will
+perform any conversions using, for example, connection settings or `convert_tz()`.
+
+Also note that only columns of `datetime` data types are converted. If you store datetime values as varchar or varbinary strings,
```
Comment (mattlord) [inline content/en/docs/14.0/reference/vreplication/movetables.md]: Here we backpack but don't capitalize. It's a minor thing, but being consistent about these things across our docs improves the overall quality.

**Review by mattlord — APPROVED** [review]
I just had a couple minor nits. Should we also take shit opportunity to update the general `MoveTables` docs page? It seems a little out of date to me: https://vitess.io/docs/14.0/concepts/move-tables/

We can do that separately if you prefer. 

Thank you!

File: content/en/docs/14.0/reference/vreplication/movetables.md (hunk 1 above)
Comment (rohit-nayak-ps) [reply content/en/docs/14.0/reference/vreplication/movetables.md]: changed

File: content/en/docs/14.0/reference/vreplication/movetables.md (hunk 2 above)
Comment (rohit-nayak-ps) [reply content/en/docs/14.0/reference/vreplication/movetables.md]: Good point, I was just lazy: made it consistent using backticks everywhere.

File: content/en/docs/14.0/reference/vreplication/movetables.md (hunk 3 above)
Comment (rohit-nayak-ps) [reply content/en/docs/14.0/reference/vreplication/movetables.md]: fixed

**Comment by mattlord** [comment]
LGTM. Thanks!

**Pull request merged**